src/main/java/frc/robot/Robot.java

```diff
@@ -120,6 +262,8 @@ public void teleopInit() {
     new DriveCommand().start();
```

Comment:
These should not be called every iteration of robotPeriodic().   You should be using the Scheduler   Scheduler.getInstance.run();